playbooks/robusta_playbooks/job_actions.py

```diff
@@ -0,0 +1,98 @@
+from robusta.api import *
+
+
+class JobParams(ActionParams):
+    """
+    :var image: The job image.
+    :var command: The job command as array of strings
+    :var name: Custom name for the job and job container.
+    :var namespace: The created job namespace.
+    :var restart_policy: Job container restart policy
+    :var job_ttl_after_finished: Delete finished job ttl (seconds). If omitted, jobs will not be deleted automatically.
+    :var notify: Add a notification for creating the job.
+    :var backoff_limit: Specifies the number of retries before marking this job failed. Defaults to 6
+    :var active_deadline_seconds: Specifies the duration in seconds relative to the startTime
+        that the job may be active before the system tries to terminate it; value must be
+        positive integer
+
+    :example command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
+    """
+    image: str
+    command: List[str]
+    name: str = "robusta-action-job"
+    namespace: str = "default"
+    restart_policy: str = "OnFailure"
+    job_ttl_after_finished: int = None
+    notify: bool = False
+    backoff_limit: int = None
+    active_deadline_seconds: int = None
+
+
+@action
+def alert_handling_job(event: PrometheusKubernetesAlert, params: JobParams):
+    """
+    Create a kubernetes job with the specified parameters
+
+    In addition, the job pod receives the following alert parameters as environment variables
+
+    ALERT_NAME
+
+    ALERT_STATUS
+
+    ALERT_OBJ_KIND - oneof pod/deployment/node/job/daemonset or None in case it's unknown
+
+    ALERT_OBJ_NAME
+
+    ALERT_OBJ_NAMESPACE (If present)
+
+    ALERT_OBJ_NODE (If present)
+
+    """
+    job_name = to_kubernetes_name(params.name)
+    job: Job = Job(
+        metadata=ObjectMeta(
+            name=job_name,
+            namespace=params.namespace
+        ),
+        spec=JobSpec(
+            template=PodTemplateSpec(
+                spec=PodSpec(
+                    containers=[
+                        Container(
+                            name=params.name,
```

Comment:
Even here `serviceAccountName` matters right?

Reply:
yes, it does matter.
Would you like to use the runner's service account, or provide one using the action parameters? Or both?